test/mix_java_scala_lib.bzl

```diff
@@ -0,0 +1,11 @@
+def _mix_java_scala_lib_impl(ctx):
+    paths = [f.path for f in ctx.files.deps]
+    for path in paths:
+        print(path)
+
+mix_java_scala_lib_rule = rule(
```

Comment:
Wdyt about mix_java_scala_lib_test_rulr? Or maybe dummy_rule?
Can live with the current one but it surprised me a bit when I first saw it in the call site

Reply:
I'm alright with changing this name. It was kind of arbitrary